openedx/adg/lms/branding_extension/tests/test_helpers.py

```diff
@@ -0,0 +1,32 @@
+from openedx.adg.lms.branding_extension.helpers import get_footer_navigation_links
+
+
+def test_get_footer_navigation_links(mocker):
+    """
+    Tests 'get_footer_navigation_links' helper method of branding extension
```

Comment:
Lets use ` instead of ' for wrapping a code block

Reply:
Done!